src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  let result = 0;
+
+  if (days >= 7) {
+    result = days * price - bigDiscount;
+
+    return result;
+  }
+
+  if (days >= 3) {
+    result = days * price - smallDiscount;
+
+    return result;
+  }
+  result = days * price;
```

Comment:
I've meant that you can move this calculation to a const before all the checks, and use it in return statements